gke/standard/regional/node_pool/main.tf

```diff
@@ -1,5 +1,5 @@
 /**
-* Copyright 2024 Google LLC
+* Copyright 2025 Google LLC
```

Comment:
We don't update the copyright headers on file updates
```suggestion
* Copyright 2024 Google LLC
```

Reply:
Reverted